README.md

```diff
@@ -1,10 +1,20 @@
 # Colouring
 
-Colouring activity [https://darts-colouring.netlify.app/](https://darts-colouring.netlify.app/)
+Netlify staging site: [https://darts-colouring.netlify.app/](https://darts-colouring.netlify.app/)
 
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## Adding New Images
 
-## Getting Started
+- Create an image in SVG format
+- Name the image as you would like its name to appear in the game menu
```

Comment:
Not sure it it's necessary but `(e.g. Flowers in a Vase.svg)`  might be helpful so the .svg is emphisised and spaces/uppercase important.